manager/manager.py

```diff
@@ -249,31 +249,23 @@ def start_job(self, job_id):
                     if error.startswith('Bad Request: '):
                         error = error[len('Bad Request: '):]
                     job.error_message(error)
-
-            if 'Payload' in json_data:
-                payload = json_data['Payload']
-
-                print('Payload:')
-                print(json.dumps(payload))
-
-                for message in payload['log']:
-                    if message:
-                        job.log_message(message)
-                for message in payload['errors']:
-                    if message:
-                        job.error_message(message)
-                for message in payload['warnings']:
-                    if message:
-                        job.warning_message(message)
-
-                success = payload['success']
-
-                if payload['errors']:
-                    job.log_message('{0} function returned with errors.'.format(module.name))
-                elif payload['warnings']:
-                    job.log_message('{0} function returned with warnings.'.format(module.name))
-                elif payload['log']:
-                    job.log_message('{0} function returned.'.format(module.name))
+                elif 'success' in json_data:
+                    success = json_data['success']
```

Comment:
I now have converter.py return "success" in the payload. I don't know how it ever got to be "status", as status = true would make no sense, yet that is what converter.py was returning: https://github.com/unfoldingWord-dev/tx-manager/pull/56/files#diff-110321eec6ff7d1d9b36706b66bb7c98R67